modules/document_repository/ajax/GetFile.php

```diff
@@ -35,27 +34,33 @@
 // Checks that config settings are set
 $config =& NDB_Config::singleton();
 
-$File = $_GET['File'];
+$file = $_GET['File'];
```

Comment:
@johnsaigle Do you want to keep using the DB approach or switch to use `basename` for consistency?